rpc_jobs/rpc_openstack.yml

```diff
@@ -142,93 +142,6 @@
     jobs:
       - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
 
-- project:
-    name: "rpc-openstack-liberty-premerge"
-    repo_name: "rpc-openstack"
-    repo_url: "https://github.com/rcbops/rpc-openstack"
-    series: "liberty"
-    branches:
-      - "liberty.*"
-    skip_pattern: |
-      \.md$
-      | \.rst$
-      | ^releasenotes/
-      | ^gating/generate_release_notes/
-      | ^gating/post_merge_test/
-      | ^gating/update_dependencies/
-    image:
-      - trusty:
-          IMAGE: "Ubuntu 14.04.5 LTS prepared for RPC deployment"
-    scenario:
-      - "ceph"
-      - "swift"
-    action:
-      - deploy:
-          FLAVOR: "performance2-15"
-    jobs:
-      - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
-
-- project:
-    name: "rpc-openstack-liberty-tox-premerge"
-    repo_name: "rpc-openstack"
-    repo_url: "https://github.com/rcbops/rpc-openstack"
-    series: "liberty"
-    branches:
-      - "liberty.*"
-    image:
-      - container:
-          SLAVE_TYPE: "container"
-    scenario:
-      - "ansible-lint"
-      - "flake8"
-    action:
-      - "tox-test"
-    jobs:
-      - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
-
-- project:
-    name: "rpc-openstack-kilo-premerge"
```

Comment:
This is a PR test, not a PM test. Are you sure you want to remove PR tests too?

Reply:
Yup. See above. Kilo isn't even installable without hacked workarounds using sed/awk, etc. We've encapsulated those in rpc-upgrade for gating purposes. No one should be installing Kilo outside of that.